source/agora/consensus/protocol/Nominator.d

```diff
@@ -122,23 +122,23 @@ extern(D):
     ***************************************************************************/
 
     public void setQuorumConfig (const ref QuorumConfig quorum,
-        const(QuorumConfig)[] other_quorums)
+        const(QuorumConfig)[] other_quorums) nothrow @safe
     {
         assert(!this.is_nominating);
-        this.quorum_set.clear();
+        () @trusted { this.quorum_set.clear(); }();
 
         // store the list of other node's quorum hashes
         foreach (qc; other_quorums)
         {
-            auto quorum_set = verifyBuildSCPConfig(qc);
+            auto quorum_set = buildSCPConfig(qc);
```

Comment:
Would it be less confusing to use a different name from the `this.quorum_set`?

Reply:
Yes, in fact the field is badly named. I will add another commit.